features/environment.py

```diff
@@ -33,13 +33,29 @@
  - chmod 755 /usr/bin/ua
 """
 
+"""
+Package: *
+Pin: release o=LP-PPA-ua-client-daily
+Pin-Priority: 1001
+"""
+UACLIENT_PPA_PIN = (
+    "UGFja2FnZTogKgpQaW46IHJlbGVhc2Ugbz1MUC1QUE"
+    "EtdWEtY2xpZW50LWRhaWx5ClBpbi1Qcmlvcml0eTogMTAwMQo="
+)
+
 USERDATA_APT_SOURCE_PPA = """\
+write_files:
+  - encoding: b64
+    content: {preference_pin}
+    owner: root:root
+    path: /etc/apt/preferences.d/uaclient
+    permissions: 0644
 apt:
   sources:
     ua-tools-ppa:
         source: deb {ppa_url} $RELEASE main
         keyid: {ppa_keyid}
-packages: [{packages}]
+packages: [openssh-server]
```

Comment:
I forgot why we need openssh-server again. I thought it was Xenial VM related in pycloudlib. Didn't we drop that image  support in pycloudlib@96b146ee1beb99b8e44e36525e18a9a20e00c3f2 ? Maybe I'm misremembering here.

Reply:
Ran both bionic and xenial VM tests without error dropping the openssh-server packages directive. I think we can avoid this since pycloudlib no longer uses  XENIAL_IMAGE_VSOCK_SUPPORT = "images:ubuntu/16.04/cloud"
which didn't include openssh-server in the image.

Reply:
Good catch. i have updated the code to remove it